opentreemap/importer/tasks.py

```diff
@@ -170,3 +172,33 @@ def _get_waiting_row_count(ie):
     return ie.rows()\
              .filter(status=GenericImportRow.WAITING)\
              .count()
+
+
```

Comment:
I think you can simplify this now, merging `species_export_builder` and `_get_export_builder` into `get_all_species_export` and removing the `all_fields` filter.